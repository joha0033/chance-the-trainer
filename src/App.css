.App-header {
  min-height: 50vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
}

.slide-appear {
  opacity    : 0;
  transition : opacity 2s ease;
}

.slide-appear.slide-appear-active {
  opacity : 1;
}

.slide-enter {
  margin-right : 0;
  transition : all .3s ease;
}

.slide-enter.slide-enter-active {
  margin-right : 500px;
}

.slide-leave {
  margin-right : 500px;
  transition : all .3s ease;
}

.slide-leave.slide-leave-active {
  margin-right : 0;
}

