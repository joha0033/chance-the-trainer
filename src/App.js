import React from 'react';
import './App.css';
import { MuiThemeProvider } from '@material-ui/core/styles';
import SimpleAppBar from './Navbar';
import theme from './theme'
import Home from './Home'
import { withStyles } from '@material-ui/core/styles';

const styles = theme => ({
  root: {
    // backgroundColor: theme.palette.primary.green,
  },
  paper: {
    // maxWidth: 500,
  },
  control: {
    // padding: theme.spacing.unit * 2,
  },
});

const App = () => {
  return (
      <MuiThemeProvider theme={theme}>
        <div className="App">
        <SimpleAppBar/>
          <header className="App-header">
          <br></br>
            <Home />
          </header>
        </div>
      </MuiThemeProvider>
      
    );
}

export default withStyles(styles)(App);
