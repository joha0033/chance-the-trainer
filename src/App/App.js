import React from 'react';
import './App.css';
import { MuiThemeProvider } from '@material-ui/core/styles';
import SimpleAppBar from '../Navbar/Navbar';
import theme from '../theme'
import Home from '../Home/Home'
import { withStyles } from '@material-ui/core/styles';
import { BrowserRouter as Router, Route } from "react-router-dom";
import Slideshow from '../Slideshow/Slideshow'

const styles = theme => ({
  root: {
    backgroundColor: theme.palette.primary.green,
  },
  App: {
    alignItems: 'center'
  },
  control: {
    // padding: theme.spacing.unit * 2,
  },
});

const App = () => {
  return (
    <Router>
       <MuiThemeProvider theme={theme}>
        <div className="App">
        <SimpleAppBar/>
          <header className="App-header">
          <br></br>
            <Route exact path='/' component={Home}/>
            <Route exact path='/Slideshow' component={Slideshow}/>
          </header>
        </div>
      </MuiThemeProvider>
    </Router>
    );
}

export default withStyles(styles)(App);
