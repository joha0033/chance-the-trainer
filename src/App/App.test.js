import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { createMemoryHistory } from 'history'

import App from './App';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});

// SNAPSHOT TESTING
it('renders correctly', () => {
  const tree = renderer
    .create(<App />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

function renderWithRouter(
  ui,
  {
    route = '/',
    history = createMemoryHistory({ initialEntries: [route] }),
  } = {}
) {
  return {
    ...render(<Router history={history}>{ui}</Router>),
    // adding `history` to the returned utilities to allow us
    // to reference it in our tests (just try to avoid using
    // this to test implementation details).
    history,
  }
}



