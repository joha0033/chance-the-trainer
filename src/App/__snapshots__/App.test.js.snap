// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <div
    className="SimpleAppBar-root-248"
  >
    <header
      className="MuiPaper-root-261 MuiPaper-elevation4-267 MuiAppBar-root-252 MuiAppBar-positionStatic-256 MuiAppBar-colorPrimary-259"
    >
      <div
        className="MuiToolbar-root-288 MuiToolbar-regular-290 MuiToolbar-gutters-289"
      >
        <div
          className="MuiGrid-container-292"
        >
          <div
            className="MuiGrid-item-293 MuiGrid-grid-xs-12-332"
          >
            <h4
              className="MuiTypography-root-389 MuiTypography-h4-404 MuiTypography-colorInherit-418 SimpleAppBar-nameLarge-251"
            >
              Chance The Trainer
            </h4>
            <h6
              className="MuiTypography-root-389 MuiTypography-subtitle2-408 MuiTypography-colorInherit-418 SimpleAppBar-nameSmall-250"
            >
              Chance The Trainer sm
            </h6>
          </div>
          <div
            className="MuiGrid-item-293 MuiGrid-grid-xs-12-332"
          >
            <h6
              className="MuiTypography-root-389 MuiTypography-h6-406 MuiTypography-colorInherit-418 SimpleAppBar-nameLarge-251"
            >
              by: ajDevs
            </h6>
            <span
              className="MuiTypography-root-389 MuiTypography-caption-399 MuiTypography-colorInherit-418 SimpleAppBar-nameSmall-250"
            >
              by: ajDevs sm
            </span>
          </div>
        </div>
        <div
          className="SimpleAppBar-demoButton-249"
        >
          <div
            className="ProgressBar"
          >
            <div>
              <label>
                <button
                  aria-label="Demo"
                  className="MuiButtonBase-root-454 MuiButton-root-428 MuiButton-text-430 MuiButton-flat-433 DemoButton-button-426"
                  disabled={false}
                  onBlur={[Function]}
                  onClick={[Function]}
                  onContextMenu={[Function]}
                  onFocus={[Function]}
                  onKeyDown={[Function]}
                  onKeyUp={[Function]}
                  onMouseDown={[Function]}
                  onMouseLeave={[Function]}
                  onMouseUp={[Function]}
                  onTouchEnd={[Function]}
                  onTouchMove={[Function]}
                  onTouchStart={[Function]}
                  tabIndex="0"
                  type="button"
                >
                  <span
                    className="MuiButton-label-429"
                  >
                    <i
                      className="material-icons DemoButton-icon-427"
                    >
                      burst_mode
                    </i>
                    Show Demo
                  </span>
                  <span
                    className="MuiTouchRipple-root-458"
                  />
                </button>
              </label>
            </div>
          </div>
        </div>
      </div>
    </header>
  </div>
  <header
    className="App-header"
  >
    <br />
    <div
      className="MuiPaper-root-261 MuiPaper-elevation5-268 MuiPaper-rounded-262 WithTheme-WithWidth-Home---root-457"
    >
      <h4
        className="MuiTypography-root-389 MuiTypography-body2-397"
      >
        <div
          className="HowToList-root-465"
        >
          <nav
            className="MuiList-root-466 MuiList-padding-467"
          >
            <h3>
              How this works
            </h3>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                   This is an app that helps with Public Speaking.
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Its a random slide show generator
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  3 slides (photos) // 30 seconds a piece
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Start a speech inspired by the photo displayed
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Relate your points from previous slide with the next
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Tie your points together with the last slide
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Click the get started button
                </span>
              </div>
            </li>
            <li
              className="MuiListItem-root-470 MuiListItem-default-473 MuiListItem-gutters-478"
              disabled={false}
              href="#simple-list"
            >
              <div
                className="MuiListItemText-root-482"
              >
                <span
                  className="MuiTypography-root-389 MuiTypography-body1-398 MuiListItemText-primary-485"
                >
                  Once directed to Slideshow, press the play button
                </span>
              </div>
            </li>
          </nav>
        </div>
        <div>
          <button
            className="MuiButtonBase-root-454 MuiButton-root-428 MuiButton-contained-439 MuiButton-containedPrimary-440 MuiButton-raised-442 MuiButton-raisedPrimary-443 GetStartedButton-button-488"
            disabled={false}
            onBlur={[Function]}
            onContextMenu={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex="0"
            type="button"
          >
            <span
              className="MuiButton-label-429"
            >
              Get Started
            </span>
            <span
              className="MuiTouchRipple-root-458"
            />
          </button>
        </div>
      </h4>
    </div>
  </header>
</div>
`;
